download.ics

```diff
@@ -0,0 +1,95 @@
+BEGIN:VCALENDAR
```

Comment:
Did you mean to include this file?

Might be worth keeping for unit tests - if so, put this into /data/test_lunch.ics